communication/zoom-guidelines.md

```diff
@@ -0,0 +1,39 @@
+# Zoom Guidelines
+
+Zoom is the main video communication platform for Kubernetes outside of our mailing lists. 
+It is used for running the [community meeting](https://github.com/kubernetes/community/blob/master/events/community-meeting.md) and SIG meetings. 
+Since the Zoom meetings are open to the general public, a Zoom host might have to moderate a meeting if a person is in violation of the code of conduct. 
```

Comment:
nit - remove "might have to". Zoom hosts should always assume responsibility of a moderator.